flink-state-backends/flink-statebackend-changelog/src/test/java/org/apache/flink/state/changelog/ChangelogStateBackendTestUtils.java

```diff
@@ -158,16 +155,15 @@ public static void testMaterializedRestore(
             keyedBackend.setCurrentKey(2);
             state.update(new StateBackendTestBase.TestPojo("u2", 2));
 
-            awaitMaterialization(keyedBackend, env.getMainMailboxExecutor());
+            // trigger materialization manually
+            periodicMaterializationManager.triggerMaterialization();
```

Comment:
The comment in code doesn't add much information IMO.

Reply:
it means each materialization is triggered manually instead of automatically schedule the next one.

Reply:
Yes, but can't one understand this from the code itself?

Reply:
It is a bit confusing unless digging into details how `triggerMaterialization()` is implemented. Because we also call schedule materialization in `triggerMaterialization()`.

Here, I mean each scheduling of materialization is triggered manually. It improves the readabilities of the test.

Reply:
My point is that comment is the same as code, it basically repeats the method name (and "manually" is implied by the presense of the call itself).

https://flink.apache.org/contributing/code-style-and-quality-common.html#comments